nativelib/src/main/scala/scala/scalanative/posix/sys/uio.scala

```diff
@@ -0,0 +1,12 @@
+package scala.scalanative
+package posix.sys
+
+import native.{CSize, Ptr, CStruct2, extern}
+
+@extern
+object uio {
+
+  // Types
+  type iovec = CStruct2[Ptr[Byte], CSize]
+
```

Comment:
`uio.h` contains two functions: `readv` and `writev`. You should include them here.

http://pubs.opengroup.org/onlinepubs/9699919799/